timeline/processing.go

```diff
@@ -915,38 +915,42 @@ func (p *processor) linkRelation(ctx context.Context, ig *Graph, tx *sql.Tx, r R
 			return fmt.Errorf("%s node: %w", fromOrTo, err)
 		}
 		if otherGraph.Item != nil {
-			if fromOrTo == relationFrom {
+			switch fromOrTo {
+			case relationFrom:
 				rawRel.fromItemID = &otherGraph.rowID.itemID
-			} else if fromOrTo == relationTo {
+			case relationTo:
 				rawRel.toItemID = &otherGraph.rowID.itemID
 			}
 		} else if otherGraph.Entity != nil {
 			attrID, err := otherGraph.rowID.identifyingAttributeID(ctx, tx)
 			if err != nil {
 				return fmt.Errorf("getting identifying attribute ID for connected entity (on %s side): %w", fromOrTo, err)
 			}
-			if fromOrTo == relationFrom {
+			switch fromOrTo {
+			case relationFrom:
 				rawRel.fromAttributeID = &attrID
-			} else if fromOrTo == relationTo {
+			case relationTo:
```

Comment:
As commented later/below, I don't like how this linter forces longer, more verbose syntax for no gain...

Reply:
I'm actually indifferent between the two, the `if`s are easier to read as single lines, but the `switch`es are easier to see as a coherent unit of "I'm doing something based on the two expected values of `fromOrTwo`.

I'm very happy to swap back to the `if` version, shall I disable/remove the linter that demands this alteration?